src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -783,6 +785,15 @@ private void DynamoView_SizeChanged(object sender, SizeChangedEventArgs e)
             {
                 fileTrustWarningPopup.UpdatePopupLocation();
             }
+
+            UpdateGeometryScalingPopupLocation();
+        }
+
+        private void UpdateGeometryScalingPopupLocation()
+        {
+            var workspaceView = this.ChildOfType<WorkspaceView>();
+            if (workspaceView != null && workspaceView.GeoScalingPopup != null)
```

Comment:
add braces.

Reply:
braces added in the next PR: https://github.com/DynamoDS/Dynamo/pull/13734